src/index.ts

```diff
@@ -127,6 +133,19 @@ export function datadog<TEvent, TResult>(
 
   const traceListener = new TraceListener(finalConfig);
 
+  // Check for duplicate installations of the Lambda library
+  detectDuplicateInstallations()
+    .then((duplicateFound) => {
```

Comment:
I put this under the `datadog()` wrapper, but I'm not sure if this is the best place to make this call. If there's somewhere that's better, I'm open to suggestions!